projects/js-packages/components/components/button/index.jsx

```diff
@@ -13,6 +13,11 @@ export const BUTTON_SIZES = {
 	SMALL: 'small',
 };
 
+export const FONT_WEIGHT = {
```

Comment:
Any reason to export it? Don't it something to be used externally, since is really sample data

Reply:
no reasons. Let's remove it.